### PR TITLE
Feature/3x3 shot

### LINKED ISSUE #4: Implement client-server communication for changing shot
Investigate and figure out how we store that information, where, and what we need to do to increment / decrement that value.

Once we know how to do that AND we have set up a basic shell to increment and decrement that value, we can begin #5.

### PR DESCRIPTION
**Pull request by jacobwilki01**
Finished the 3x3 Shot.

### CODE REVIEW

File: back/server/assets/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "DevServerAddress": "http://127.0.0.1:5100",
+    "DevServerAddress": "http://127.0.0.1:5101",
```
Comment (modmo22) [inline back/server/assets/config.json]: What's the reason for the port number change? Was it just an arbitrary change you made to test on a different port? This should be changed back to 5100 to align with the env documentation in the README files in the project root and root/back dirs or the READMEs need to be edited to reflect the change.

File: front/assets/config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "DevServerAddress": "http://127.0.0.1:5100",
+    "DevServerAddress": "http://127.0.0.1:5101",
```
Comment (modmo22) [inline front/assets/config.json]: Change this back to 5100 or update README files.

File: front/assets/config_dev.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "DevServerAddress": "http://127.0.0.1:5100",
+  "DevServerAddress": "http://127.0.0.1:5101",
```
Comment (modmo22) [inline front/assets/config_dev.json]: Same here.

**Review by richardmoser — APPROVED** [review]

**Review by modmo22 — CHANGES_REQUESTED** [review]
I think the config files should be added to .gitignore. That way if devs want to mess around with port numbers, when the clientId changes due to testing locally, etc. we won't have to constantly ensure these values are set to their default.

If the config files get added to .gitignore, then the rest of my changes can be ignored.

File: back/server/assets/config.json (hunk 1 above)
Comment (jacobwilki01) [reply back/server/assets/config.json]: My VSCode crashed and I had a phantom server app running I couldn't stop, so I swapped the port. I'll run npm run clean.

**Review by modmo22 — APPROVED** [review]

**Pull request merged**